cluster/gce/util.sh

```diff
@@ -742,44 +742,14 @@ function create-etcd-certs {
   local ca_cert=${2:-}
   local ca_key=${3:-}
 
-  download-cfssl
+  GEN_ETCD_CA_CERT="${ca_cert}" GEN_ETCD_CA_KEY="${ca_key}" \
+    generate-etcd-cert "${KUBE_TEMP}/cfssl" "${host}" "peer" "peer"
```

Comment:
The directory `${KUBE_TEMP}/cfssl` will create in `download-cfssl` function, but it's not exists before call download in `generate-etcd-cert` function. It maybe cause error: `the file or directory not exists`. I'll fix it.